modules/vaos/spec/services/ccra/referral_service_spec.rb

```diff
@@ -5,17 +5,15 @@
 describe Ccra::ReferralService do
   subject { described_class.new(user) }
 
-  let(:user) { double('User', account_uuid: '1234', flipper_id: '1234') }
-  let(:access_token) { 'fake-access-token' }
+  let(:user) { double('User', account_uuid: '1234', flipper_id: '1234', icn: '1012845331V153043') }
```

Comment:
Note: Only had to change the unit test setup. The logic for the tests stays the same.